packages/features/tasker/tasks/triggerNoShow/schema.ts

```diff
@@ -7,6 +7,8 @@ const commonSchema = z.object({
   triggerEvent: z.enum([
     WebhookTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW,
     WebhookTriggerEvents.AFTER_GUESTS_CAL_VIDEO_NO_SHOW,
+    WebhookTriggerEvents.AFTER_HOSTS_CALL_NO_SHOW,
+    WebhookTriggerEvents.AFTER_GUESTS_CALL_NO_SHOW,
```

Comment:
Created this common trigger

Reply:
I'll remove WebhookTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW,
    WebhookTriggerEvents.AFTER_GUESTS_CAL_VIDEO_NO_SHOW in a follow up PR without breaking